manifests/trusted_external_command.pp

```diff
@@ -61,6 +65,7 @@
         instance          => $instance,
         user              => $user,
         password          => $password,
+        oauth_token       => $oauth_token,
```

Comment:
Also, the spec test for this class should _not_ be deleted. I've found that even simple `it compiles` tests were really useful in detecting errors like a typo in the `.epp` file. (I couldn't put this comment in the spec test file since it was deleted).